app/src/main/java/com/example/todolist/MainActivity.java

```diff
@@ -67,14 +104,28 @@ public class MainActivity extends AppCompatActivity implements ToDoClickListener
     private boolean[] toTop = new boolean[2];
     private int toTopControl = 0; // 0: controlling incomplete list; 1: controlling completed list
 
+
     // initialization code
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+
+        //Creates the notification channel that can be toggled on in the app info settings - Default is off.
+        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O)
+        {
+            NotificationChannel channel = new NotificationChannel("NotifyLate", "Late Task Notification", NotificationManager.IMPORTANCE_HIGH);
+
+            NotificationManager manager = getSystemService(NotificationManager.class);
+            manager.createNotificationChannel(channel);
+        }
+
+        recyclerView = findViewById(R.id.recyclerView);
```

Comment:
This is an error. No such variable "recyclerView". There is also no component "R.id.recyclerView". Seems that just removing it fixes it and causes no problems

Reply:
kk removed that line, I wasnt getting that error on myside. let me know if that corrected it for you!